packages/components/src/styles/components/table.scss

```diff
@@ -225,7 +210,10 @@ $hds-table-cell-padding-tall: 22px 16px 21px 16px; // the 1px difference is to a
     .hds-table--density-tall & {
       padding: $hds-table-cell-padding-tall;
     }
+  }
 
+  .hds-table__td:last-child {
+    border-right: $hds-table-border-width solid $hds-table-border-color;
```

Comment:
Would something like this work for the border styling? (I don't know how well it would work cross-browser though or with the table grid structure...)

```
.hds-table, 
.hds-table__th, 
.hds-table__td {
    border: calc(#{$hds-table-border-width} / 2) solid $hds-table-border-color;
}
```

Reply:
Yeah, I tried that originally and it looks fine in Safari but in Chrome it results in a 2px border.

![Uploading Screenshot 2024-12-16 at 10.50.51 AM.png…]()

Reply:
![Screenshot 2024-12-16 at 10 50 51 AM](https://github.com/user-attachments/assets/e5f99e3b-c86c-45d3-8d8e-04ce6fe31a72)

Reply:
Updated: testing more, if I do border-collapse: collapse, it works as expected across browsers - but the border-radius is lost.

Reply:
Thanks for testing. I tested just by editing within the Chrome browser. I first unchecked all the existing border styles then added back a .5px border to everything. I compared with another tab in which I left the styles unedited and the borders seemed to be the same width.

<img width="1165" alt="image" src="https://github.com/user-attachments/assets/5de9b690-36fc-4388-8611-05ad2df3debe" />

Reply:
Funny! The borders also look incorrect in percy: https://percy.io/HashiCorp/web/hds-components/builds/37844826/changed/2058208695?browser=chrome&browser_ids=59&group_snapshots_by=similar_diff&subcategories=unreviewed%2Cchanges_requested&utm_source=github_status_private&viewLayout=overlay&viewMode=original&width=1280&widths=375%2C1280

Reply:
It does cause a slight shift of some borders but it doesn't seem to effect the width of them.

I created a simplified test and checked in Chrome, Safari, Firefox, & Edge and applying .5px width borders seemed to work well in all: https://codepen.io/KristinB/pen/NPKdgxv/f5c2ee448f0e40e60d806252b57a9691

Reply:
Just to wrap up this thread: it turns out it is because of my display. We are moving forward with this solution because the potential of a thicker border is worth the simpler/more flexible implementation.